jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -10,36 +10,72 @@
 from astropy.modeling.models import Gaussian1D
 from astropy.time import Time
 from glue.core.message import SubsetUpdateMessage
+from glue_jupyter.common.toolbar_vuetify import read_icon
 from ipywidgets import widget_serialization
 from packaging.version import Version
 from photutils.aperture import (ApertureStats, CircularAperture, EllipticalAperture,
                                 RectangularAperture)
 from traitlets import Any, Bool, Integer, List, Unicode, observe
 
+from jdaviz.core.custom_traitlets import FloatHandleEmpty
 from jdaviz.core.events import SnackbarMessage, LinkUpdatedMessage
 from jdaviz.core.region_translators import regions2aperture, _get_region_from_spatial_subset
 from jdaviz.core.registries import tray_registry
-from jdaviz.core.template_mixin import (PluginTemplateMixin, DatasetSelectMixin,
+from jdaviz.core.template_mixin import (PluginTemplateMixin, DatasetMultiSelectMixin,
                                         SubsetSelect, TableMixin, PlotMixin)
+from jdaviz.core.tools import ICON_DIR
 from jdaviz.utils import PRIHDR_KEY
 
 __all__ = ['SimpleAperturePhotometry']
 
 ASTROPY_LT_5_2 = Version(astropy.__version__) < Version('5.2')
 
 
-@tray_registry('imviz-aper-phot-simple', label="Imviz Simple Aperture Photometry")
-class SimpleAperturePhotometry(PluginTemplateMixin, DatasetSelectMixin, TableMixin, PlotMixin):
+@tray_registry('imviz-aper-phot-simple', label="Aperture Photometry")
+class SimpleAperturePhotometry(PluginTemplateMixin, DatasetMultiSelectMixin, TableMixin, PlotMixin):
+    """
+    The Aperture Photometry plugin performs aperture photometry for drawn regions.
+    See the :ref:`Aperture Photometry Plugin Documentation <aper-phot-simple>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``dataset`` (:class:`~jdaviz.core.template_mixin.DatasetSelect`):
+      Dataset to use for photometry.
+    * ``aperture`` (:class:`~jdaviz.core.template_mixin.SubsetSelect`):
+      Subset to use as the aperture.
+    * ``background`` (:class:`~jdaviz.core.template_mixin.SubsetSelect`):
+      Subset to use to calculate the background.
+    * ``background_value``:
+      Background to subtract, same unit as data.  Automatically computed if ``background``
+      is set to a subset.
+    * ``pixel_area``:
+      Pixel area in arcsec squared, only used if sr in data unit.
+    * ``counts_factor``:
+      Factor to convert data unit to counts, in unit of flux/counts.
+    * ``flux_scaling``:
+      Same unit as data, used in -2.5 * log(flux / flux_scaling).
+    * :meth:`calculate_photometry`
+    * :meth:`unpack_batch_options`
+    * :meth:`calculate_batch_photometry`

```

Comment:
Since these are no longer public in this PR, we can also remove them from the docstring. Maybe turn them into in-line comments like how you did it with the equivalent API stuff?

```suggestion
```